aviary/subsystems/mass/flops_based/wing_detailed.py

```diff
@@ -212,8 +218,12 @@ def compute(self, inputs, outputs):
 
                 delme = dy * eel[1:]
 
-                delme[loc[-1]] = engine_locations[idx:idx2][0] - \
-                    integration_stations[loc[-1]]
+                if num_wing_engines[i] > 0:
+                    delme[loc[-1]] = engine_locations[idx:idx2][0] - \
+                        integration_stations[loc[-1]]
+                else:
```

Comment:
Probably don't need the "else" branch anymore, since num_wing_engines=0 bails before it reaches this code now.